testutils/check-calling-form-in-baseline/main.go

```diff
@@ -175,45 +176,78 @@ type checkConfig struct {
 	form string
 }
 
-var (
-	callingFormRe = regexp.MustCompile(`calling form: "(.*?)"`)
-	valueSetRe    = regexp.MustCompile(`valueSet "(.*?)"`)
-)
-
 // deleteFoundForms reads the baseline, then deletes any used calling forms and value sets from forms.
-func deleteFoundForms(baseline string, lang string, forms map[checkConfig]bool) error {
-	// Split the baseline file into its "logical files"
-	var files []string
-	for {
-		const fileHeader = "============== file:"
-		p := strings.Index(baseline, fileHeader)
-		if p < 0 {
-			files = append(files, baseline)
-			break
-		}
-		files = append(files, baseline[:p])
+func deleteFoundForms(r io.Reader, lang string, forms map[checkConfig]bool) error {
+	var callingForm, valueSet string
```

Comment:
Seems like just having a `var formInstance checkConfig` instead would be more succinct?